site/cds_rdm/ext.py

```diff
@@ -32,6 +33,7 @@ def init_app(self, app):
         """Flask application initialization."""
         self.init_services(app)
         self.init_resources(app)
+        app.jinja_env.globals["get_clc_sync_entry"] = get_clc_sync_entry
```

Comment:
in this way you are adding it to all the templates in the whole application, do we really want this ? as well as I think we are accessing the db for all the records, does it make sense to check only for specific resource types? otherwise you always need to check if record has sync entry, even though it will not be the case for the majority of them

Reply:
> do we really want this?

Yes, unless you have better ideas. This is an easy way to get some date from the backend, without changing the view and inject more context.

> does it make sense to check only for specific resource types? 

Yes why not, it can be done in the service so that we can skip the call if not matching the resource type. @kpsherva do we know the resource types? @jrcastro2 WDYT?

Reply:
anything that can exist in a paper form, so all publications but if you want to specify even more:
```
publication-book
publication-bookchapter
publication-conferencepaper
publication-conferenceproceedings
publication-journal
publication-journalarticle
publication-preprint
publication-report
publication-technicalnote
publication-thesis
publication-dissertation
publication-workingpaper
```

Reply:
I adde da check for any publication. Let me know if that is good enough or if we should add specifically the ones you have shared here.